en/about/website/index.md

```diff
@@ -68,7 +68,7 @@ Also many thanks to the organizations that support us:
 
 <img src="../../../images/sponsor/dd.png" alt="Datadog" width="200" height="200" />
 
-[1password](https://1password.com/) (password manager)
+[1Password](1password) (password manager)
```

Comment:
Should this be `[1Password][1password]`?

Reply:
Ah, yes. I fixed at https://github.com/ruby/www.ruby-lang.org/commit/88ad266088656c9bd8f4f5a7711ec2f4830f151b

Thanks!